source/infrastructure/deploy-core-single-stack.bash

```diff
@@ -284,6 +286,10 @@ if [ -n "$USE_EXISTING_VPC" ] && [ "$USE_EXISTING_VPC" = 'true' ]; then
     DEPLOY_PARAMETERS+=( ExistingPrivateRouteTableIds=$PRIVATE_ROUTE_TABLE_IDS )
 fi
 
+if [ -n "$NEPTUNE_INSTANCE_TYPE" ]; then
```

Comment:
we are checking if the string isn't empty. In the case if it is empty, i would assume an empty string will be passed to the cfn template which would defaulted at the cfn parameter level. Have you verified different scenarios ? to make sure we are handling edge cases for single-stack deployment ?

Reply:
I haven't tested with the single-stack deployment. We use the normal deploy-core.bash and I've tested with that deployment a few different ways. It uses a similar approach.

Reply:
If we pass don't specify the param in the `aws cloudformation deploy` command, it will default to the existing param in the stack (for stack updates) or the default in the CFN (for a new stack).

Reply:
is it possible for you to verify the single stack deployment ?

Reply:
OK - I did a little testing (sorry it took me a few days to get some time to test). Deploying a new single-stack, not passing in any parameters successfully defaults to db.r4.xlarge.
<img width="658" alt="no-passed-in-parameter" src="https://user-images.githubusercontent.com/89797921/135649368-e6f36fdb-985f-486c-9ab5-ce71556d4306.png">

Also, passing in db.t3.medium successfully sets the parameter
<img width="605" alt="pass-in-db t3 medium" src="https://user-images.githubusercontent.com/89797921/135649366-79432c4e-6c27-46c7-bab1-7ecc8127df79.png">